doc_source/private-clusters.md

```diff
@@ -26,6 +26,7 @@ Here are some things to consider when running Amazon EKS in a private cluster wi
 + The [Amazon EFS CSI driver](efs-csi.md) is supported\. Before deploying, the [kustomization\.yaml](https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/master/deploy/kubernetes/overlays/stable/kustomization.yaml) file must be changed to set the container images to use the same Region as the Amazon EKS cluster\.
 + The [Amazon FSx for Lustre CSI driver](fsx-csi.md) is not supported\.
 + [AWS Fargate](fargate.md) is supported with private clusters\. You must include the STS VPC endpoint\. For more information, see [VPC endpoints for private clusters](#vpc-endpoints-private-clusters)\. You can use the AWS Load Balancer Controller to deploy AWS Application Load Balancers and Network Load Balancers with\. The controller supports network load balancers with IP targets, which are required for use with Fargate\. For more information, see [Application load balancing on Amazon EKS](alb-ingress.md) and [Create a network load balancer](network-load-balancing.md#network-load-balancer)\.
++ [AWS Loadbalancer Controller](https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html) is supported, However while installing you should disable default true configurations such as enable-shield, enable-waf, enable-wafv2 using [Flags](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.2/deploy/configurations/#controller-command-line-flags). In addition, [autodiscovery of certificate](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.1/guide/ingress/cert_discovery/#discover-via-ingress-rule-host) in the ingress objects based on hostname is not supported as the controller needs to reach ACM which doesnt have vpc endpoint.
```

Comment:
Thanks for contribution. the content lgtm with two nit comment below:
`[Flag]` -> `[command line flags]`
`[autodiscovery of certificate]` -> `[certificate discovery]`